packages/explorer/src/common.ts

```diff
@@ -1,21 +1,23 @@
 import { anvil, garnet, redstone } from "viem/chains";
 
 export const supportedChains = { anvil, garnet, redstone } as const;
-export const supportedChainsById = Object.fromEntries(
-  Object.entries(supportedChains).map(([, chain]) => [chain.id, chain]),
-);
+export type supportedChains = typeof supportedChains;
```

Comment:
Wonder if it should start uppercase given it's a type, and that's been the convention so far?

Reply:
this is just a shortcut to avoid having to sprinkle `typeof` everywhere you're using it with a `const`

when the name mirrors the type, you can use the same name/reference in both types and runtime